README.md

```diff
@@ -17,10 +17,16 @@ Add the `loco-openapi` initializer, with one or multiple of the following featur
 - `scalar`
 - `full`
 
+Temporarily (before loco 15.1):
+Change the loco-rs to the master branch.
```

Comment:
I prefer to remove it and include a note stating that this is supported starting from Loco version 0.16 and above.